portfolio/src/main/webapp/hobbies.html

```diff
@@ -9,6 +9,125 @@
   <body>
     <div id="content">
       <h1>My Hobbies</h1>
+      <h3>Here is a list of some of my many hobbies (in no particular order):
+      </h3>
+      <ul>
+        <li><a href="#ballroom">Ballroom and latin dancing</a></li>
+        <li><a href="#xstitch">Cross stitching</a></li>
+        <li>Knitting</li>
```

Comment:
Will you be adding links for Knitting and the other lists as well? If so , add a TODO comment

https://google.github.io/styleguide/htmlcssguide.html#Action_Items

Reply:
I wasn't necessarily planning on it, but I could if I have time. Would you still advise adding a TODO if it's a stretch goal?

Reply:
In that case, don't add the TODO :)